pkg/operator/operator.go

```diff
@@ -563,12 +570,12 @@ func (o *Operator) sync(ctx context.Context, key string) error {
 	return nil
 }
 
-func (o *Operator) reportError(ctx context.Context, err error, taskName string) {
-	klog.Infof("ClusterOperator reconciliation failed (attempt %d), retrying. Err: %v", o.failedReconcileAttempts+1, err)
-	if o.failedReconcileAttempts == 2 {
+func (o *Operator) reportError(ctx context.Context, err error, failedTaskReason string) {
+	klog.Infof("ClusterOperator reconciliation failed (attempt %d), retrying. ", o.failedReconcileAttempts+1)
+	if o.failedReconcileAttempts >= 2 {
```

Comment:
since the CMO is reconciled every 5 minutes, then we should wait for at least 10 minutes to see the failed info in 
`oc get co monitoring -oyaml`, I think the time is a little long, WDYT?

Reply:
This is to avoid CMO being reported as degraded when there are other minor issues, by trying twice we can avoid this.
Once it is degraded, then it will report failure on every instance.